src/go/rpk/pkg/cli/cmd/topic_test.go

```diff
@@ -177,11 +177,11 @@ func TestTopicCmd(t *testing.T) {
 				},
 			},
 			args: []string{},
-			expectedOutput: `  Name       Partitions  Replicas
-  fukushima  7           3
-  kyoto      10          2
-  tokyo      2           3
-`,
+			expectedOutput: `  Name       Partitions  Replicas  ` + "\n" +
+				`  fukushima  7           3         ` + "\n" +
+				`  kyoto      10          2         ` + "\n" +
+				`  tokyo      2           3         ` + "\n" +
+				``,
```

Comment:
@0x5d I did this ugly hack in order to pass the tests. 
I think there could be another solution to that problem. 

1. Split output by new line
2. Run `strings.TrimSpace(line)` for `expectedOutput` and `out.String()`
3. Compere each line.

Reply:
I don't know why, but now everything works as expected without this change.

Reply:
They're probably using more sensible simplication now :) https://github.com/cockroachdb/crlfmt/commit/cae1965cbdf28291a28715f9b91a7a4061c1c8ed